src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+
+    <form
```

Comment:
Remove unnecessary  spaces 
```suggestion
    <form
```